Doc/library/ctypes.rst

```diff
@@ -1799,10 +1793,14 @@ different ways, depending on the type and number of the parameters in the call:
    integer. *name* is name of the COM method. *iid* is an optional pointer to
    the interface identifier which is used in extended error reporting.
 
+   If *iid* is not specified, an :exc:`OSError` is raised if the COM method
+   call fails. If *iid* is specified, a :exc:`.COMError` is raised instead.
```

Comment:
I think (but I'm not 100% sure) that `.COMError` means Sphinx will search *all* namespaces, not just `ctypes`, for a `COMError`. So it would be better to use `ctypes.COMError` explicitly:
```suggestion
   call fails. If *iid* is specified, a :exc:`~ctypes.COMError` is raised instead.
```